server/src/main/java/io/seata/server/storage/db/store/LogStoreDataBaseDAO.java

```diff
@@ -165,15 +164,9 @@ public GlobalTransactionDO queryGlobalTransactionDO(long transactionId) {
             conn = logStoreDataSource.getConnection();
             conn.setAutoCommit(true);
 
-            StringBuilder sb = new StringBuilder();
-            for (int i = 0; i < statuses.length; i++) {
-                sb.append("?");
-                if (i != (statuses.length - 1)) {
-                    sb.append(", ");
-                }
-            }
+            String paramsPlaceHolder = org.apache.commons.lang.StringUtils.repeat("?",",",statuses.length);
```

Comment:
extract `repeat` method to io.seata.common.util.StringUtils?

Reply:
I don't think it's necessary.

Reply:
the `apache common langs` jar often conflicts with other libraries. convenient direct replacement.